models/games.models.js

```diff
@@ -11,9 +11,24 @@ exports.selectReviewId = (review_id) => {
   return connection
     .query("SELECT * FROM reviews WHERE review_id = $1", [review_id])
     .then((result) => {
-      if (result.rows.length === 0){
-        return Promise.reject({ status:404, msg: "not found"})
+      if (result.rows.length === 0) {
+        return Promise.reject({ status: 404, msg: "not found" });
       }
-      return result.rows[0]
+      return result.rows[0];
     });
 };
+
+exports.selectReviews = (sort_by = "created_at DESC") => {
+  return connection
+  .query(`
+  SELECT reviews.*, COUNT(comments.comment_id) AS comment_count FROM reviews LEFT JOIN comments ON reviews.review_id = comments.review_id GROUP BY reviews.review_id ORDER BY ${sort_by};
```

Comment:
Using template literals %{} opens us up to SQL injection here. However the ticket does not require the sorting feature to be dynamic, so this can be hard coded.